src/App.jsx

```diff
@@ -0,0 +1,53 @@
+import { useState, useEffect } from "react";
+import "./styles.css";
+import Task from "./components/Task";
+
+function App() {
+  const [newItem, setNewItem] = useState("");
+  const [newTask, setnewTask] = useState(() => {
+    const storedTasks = localStorage.getItem('TASKS');
+    if (storedTasks == null) return [];
+
+    return JSON.parse(storedTasks);
+  });
+
+  useEffect(() => {
+    localStorage.setItem("TASKS", JSON.stringify(newTask));
+  }, [newTask])
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    setnewTask((currentTasks) => {
+      return [...currentTasks, { id: crypto.randomUUID(), title: newItem, completed: false }
+      ]
+    });
+
+    setNewItem("");
+  }
```

Comment:
It would be better to check if the newItem is empty or not and if it is not empty add it